src/wovnio/wovnphp/Url.php

```diff
@@ -99,6 +104,14 @@ public static function addLangCode($uri, $store, $lang, $headers)
                 case 'query':
                     $new_uri = self::addQueryLangCode($no_lang_uri, $lang_code, $lang_param_name);
                     break;
+                case 'custom_domain':
+                    if (self::isAbsolutePath($no_lang_uri)) { // absolute path
```

Comment:
WOVN.php relative path handling is pretty broken so this is OK for now. The correct way would be converting relative paths to absolute, but this is likely not easy in PHP.